hub/src/server/authentication.ts

```diff
@@ -126,11 +128,16 @@ export class V1Authentication {
     // associationToken and verifies that it authorizes the "outer"
     // JWT's address (`bearerAddress`)
 
-    let associationToken: TokenType
-    try {
-      associationToken = decodeToken(token)
-    } catch (e) {
-      throw new ValidationError('Failed to decode association token in JWT')
+    const associationToken = (() => {
```

Comment:
Should this snippit -- the `decodeToken(token)`, its try/catch block, and the subsequent `typeof` check -- be folded into its own method?  I see it's used multiple times.

Reply:
Fixed in https://github.com/blockstack/gaia/pull/245/commits/a792184381884306db0c47ee6b4c2b7d24856213
(This PR will have all package update related diffs removed once that PR is merged).